### PR TITLE
added ANET A6 with BLTouch example configuration

### PR DESCRIPTION
**Pull request by ralf-e**

### CODE REVIEW

**Comment by thinkyhead** [comment]
If this doesn't differ much from the Anet config, then perhaps it doesn't need to be an entirely separate configuration. Instead, just set the proper parameters for a BLTOUCH probe in the existing Anet configuration, then add a comment on the BLTOUCH option, or at the top of the configuration, that there is a model of Anet that comes with a BLTouch probe.

**Comment by ralf-e** [comment]
The main reason for this config was not the BL-touch but the printer type (ANET A6). From Anet there are several printers with different mechanics (different path lengths, other Z-axis, more stable). The already existing exampleconfig from Bob would have to be for the ANET A8. The BLTouch is just as an example in it, you could also take out for a somewhat general config.

**Comment by benlye** [comment]
Is it worth adding a Readme.md file for Anet, like some of the other example configurations have?

**Comment by thinkyhead** [comment]
I see! So, if I moved the other one to `Anet/A8` and this one into `Anet/A6` then we'll have configs for both? Sweet!

File: Marlin/example_configurations/Anet/ANET A6 with BLTouch/Configuration_adv.h [hunk 1]
```diff
@@ -0,0 +1,1366 @@
...
+   * but only if the current temperature is far enough below the target for a reliable test.
+   *
+   * If you get false positives for "Heating failed" increase WATCH_TEMP_PERIOD and/or decrease WATCH_TEMP_INCREASE
+   * WATCH_TEMP_INCREASE should not be below 2.
+   */
+  //#define WATCH_TEMP_PERIOD 20                // Seconds
+  //#define WATCH_TEMP_INCREASE 2               // Degrees Celsius
+  //ANET A6:
+  #define THERMAL_PROTECTION_BED_PERIOD 60    // Seconds
+  #define THERMAL_PROTECTION_BED_HYSTERESIS 5 // Degrees Celsius
```
Comment (thinkyhead) [inline Marlin/example_configurations/Anet/ANET A6 with BLTouch/Configuration_adv.h]: Oops! What did you mean to set here?

Comment (ralf-e) [reply Marlin/example_configurations/Anet/ANET A6 with BLTouch/Configuration_adv.h]: Oops :-) nothing.  Copy-Paste-Error (from my old Skynet-Setting)

**Comment by thinkyhead** [comment]
Ready to merge at #7179

**Pull request closed without merging**